CHANGELOG.md

```diff
@@ -2,12 +2,19 @@
 
 ### Added
 
+- `Dialog`: added `leftAction` property ([@driesd](https://github.com/driesd) in [#1940](https://github.com/teamleadercrm/ui/pull/1940))
+
 ### Changed
 
+- `Dialog`: changed `tertiaryAction` from a `Link` to a `Button` component ([@driesd](https://github.com/driesd) in [#1940](https://github.com/teamleadercrm/ui/pull/1940))
+- 
```

Comment:
These PR references don't seem right 🤔 (1940)

Reply:
Will correct this